README.md

```diff
@@ -47,10 +47,11 @@ pushed to your docker repository.
 
 Use the appropriate make targets or scripts in `hack`:
 
-- `make dev`: Deploys the serverless-operator without deploying Knative Serving and Eventing.
-- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving and Eventing.
+- `make dev`: Deploys the serverless-operator without deploying Knative Serving, Eventing and Kafka components.
+- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving, Eventing and Kafka components.
```

Comment:
Is this accurate? Seems like kafka would not be installed by this.

Reply:
This is fixed now